minio/umbrel-app.yml

```
@@ -0,0 +1,34 @@
+manifestVersion: 1.1
+id: minio
+category: developer
+name: MinIO
+version: "RELEASE.2024-02-17T01-15-57Z"
+tagline: The Object Store for AI Data Infrastructure
+description: >-
+    MinIO is a high-performance, S3 compatible object store. It is built for
+    large scale AI/ML, data lake and database workloads. It is software-defined
+    and runs on any cloud or on-premises infrastructure. MinIO is dual-licensed
+    under open source GNU AGPL v3 and a commercial enterprise license.
+
+    Understand the Features that Make MinIO the Industry's Most Widely Deployed Object Store:
+    - Bucket & Object Immutability
+    - Encryption
+    - Active Active Replication for Object Storage
+    - Automated Data Management Interfaces
+    - Object Storage Built for S3
+    - Data Life Cycle Management & Tiering
+    - Scalable Object Storage
+    - Identity & Access Management
+developer: "MinIO, Inc."
+website: https://min.io
+dependencies: []
+repo: https://github.com/minio/minio
+support: https://slack.min.io
+port: 9011
+gallery:
+    - 1.jpg
+    - 2.jpg
+    - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```

This will show the user the default username and password in the app store after they download the app and when they right-click on an app on the homescreen and select "show default credentials"

```suggestion
defaultUsername: "umbrel"
deterministicPassword: true
```